secretmanager/runtime/src/main/java/io/quarkiverse/googlecloudservices/secretmanager/runtime/SecretManagerProvider.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkiverse.googlecloudservices.secretmanager.runtime;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import io.quarkiverse.googlecloudservices.common.GcpConfiguration;
+import io.quarkus.credentials.CredentialsProvider;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+public class SecretManagerProvider implements CredentialsProvider {
```

Comment:
This is a `CredentialsProvider` so better named it `SecretManagerCredentialsProvider`.

Reply:
Renamed